engine/runtime-instrument-common/src/main/scala/org/enso/interpreter/instrument/job/UpsertVisualizationJob.scala

```diff
@@ -70,11 +71,19 @@ class UpsertVisualizationJob(
 
       maybeCallable match {
         case Left(ModuleNotFound(moduleName)) =>
-          replyWithError(Api.ModuleNotFound(moduleName))
+          ctx.endpoint.sendToClient(
+            Api.Response(Api.ModuleNotFound(moduleName))
+          )
           None
 
         case Left(EvaluationFailed(message, result)) =>
-          replyWithExpressionFailedError(message, result)
+          replyWithExpressionFailedError(
```

Comment:
Those should actually be changed to be sent as a notification now, rather than a response.